### PR TITLE
Prevent the same trivia question being sent twice

### LINKED ISSUE #14: Questions get sent twice in a row.
#The Issue:
While playing trivia its possible that you can see the same question more the once in a game.

#Proposed Solution:
Add each question to an array as it's sent. Then before sending a new one make sure that it isn't already in the array. Reset the array after each game or after `x` amount of questions (most likely somewhere around 25)

### PR DESCRIPTION
**Pull request by jeevanshah07**
There are times when the same trivia questions will be sent twice in a game, this fixed that. 
Closes #14

### CODE REVIEW

File: src/cogs/trivia.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 locked_channels = []  # Channel IDs that are currently in use by a game
 
+questions = []  # Questions that have been sent
```
Comment (TheOtherUnknown) [inline src/cogs/trivia.py]: Given this scope, won't this affect *all* games?

Comment (jeevanshah07) [reply src/cogs/trivia.py]: Fixed

**Comment by TheOtherUnknown** [comment]
This looks good, except for one small problem: If the bot runs out of questions, it crashes. That's not great. Some sorta message would be better.

**Comment by jeevanshah07** [comment]
> This looks good, except for one small problem: If the bot runs out of questions, it crashes. That's not great. Some sorta message would be better. 

I'm pretty sure I have it set so that if there have been 100 unique questions sent then it will start resending questions. I'm not totally sure and I'm on vacation for my spring break so I can double check once I get home.

**Review by TheOtherUnknown — APPROVED** [review]

**Pull request merged**